docs/runbook.md

```diff
@@ -0,0 +1,245 @@
+Runbook
+=======
+
+For regular operations and maintenance of the Router back-end of the Console
+
+## Once Per New Server
+
+General installation instructions are available on the Helium documentation
+website.
+
+> Start here:
+[Run a Network Server](https://docs.helium.com/use-the-network/run-a-network-server/)
+which goes from `git clone` through to various initial `docker` commands
+such as loading an initial snapshot and testing with a real or virtual device.
+
+Syntax of **Bash** shell is used below.
+
+Create shell aliases.
+
+Router:
+
+```bash
+alias router='docker exec -it helium_router router'
+alias wallet='~/helium-wallet-rs/target/release/helium-wallet'
+```
+
+Console:
+
+```bash
+alias console_down='docker-compose down'
+alias console_psql='docker exec -it helium_postgres psql -U postgres'
+alias console_restart='console_down && console_up'
+alias console_up='docker-compose build && docker-compose up -d && docker-compose logs -f'
+alias console_update='docker-compose build --pull && console_restart'
+```
+
+Miscellaneous:
+
+```bash
+alias alert='notify-send --urgency=low -i "$([ $? = 0 ] && echo terminal || echo error)" "$(history|tail -n1|sed -e '\''s/^\s*[0-9]\+\s*//;s/[;&|]\s*alert$//'\'')"'
```

Comment:
I think this alias comes default on ec2 instances. We haven't used it for anything.

Reply:
Removed with commit 7e102c6